ckan/tests/helpers.py

```diff
@@ -213,6 +214,12 @@ def submit_and_follow(app, form, extra_environ=None, name=None,
     Call webtest_submit with name/value passed expecting a redirect
     and return the response from following that redirect.
     '''
+    if extra_environ is None:
+        extra_environ = {}
+    if not extra_environ.get('HTTP_HOST'):
+        extra_environ['HTTP_HOST'] = str(
+            urlparse.urlparse(config['ckan.site_url']).netloc)
+
```

Comment:
@amercader When I remove this change the tests are still passing for me, what am I missing?